plugin/ipam/cni.go

```diff
@@ -71,9 +80,10 @@ func (i *Ipam) Release(args *skel.CmdArgs) error {
 }
 
 type ipamConf struct {
-	Subnet  string         `json:"subnet,omitempty"`
-	Gateway net.IP         `json:"gateway,omitempty"`
-	Routes  []*types.Route `json:"routes"`
+	Subnet  string              `json:"subnet,omitempty"`
+	Gateway net.IP              `json:"gateway,omitempty"`
+	Routes  []*types.Route      `json:"routes"`
+	IPs     []*current.IPConfig `json:"ips,omitempty"`
```

Comment:
Per the [Conventions](https://github.com/containernetworking/cni/blob/master/CONVENTIONS.md#well-known-capabilities) this should be "string entries". Maybe you can find a plugin at https://github.com/containernetworking/plugins that implements `ips` and try to be consistent with that?